src/server/sams/api/consume/assets.py

```diff
@@ -50,6 +53,30 @@ def download_binary(asset_id):
     return response
 
 
+@assets_bp.route('/consume/assets/compressed_binary/<asset_ids>', methods=['GET'])
+def download_compressed_binary(asset_ids):
+    """
+    Uses asset_ids and returns the compressed
+    asset binaries zip
+    """
+    asset_ids = ast.literal_eval(asset_ids) if asset_ids else None
+    file = [get_asset_service().download_binary(asset_id) for asset_id in asset_ids]
+    files = [(single_file.filename, single_file.read()) for single_file in file]
+
+    in_memory_zip = BytesIO()
+    with zipfile.ZipFile(in_memory_zip, mode='w') as temp_zip:
+        for f in files:
+            temp_zip.writestr(f[0], f[1])
+
+    response = app.response_class(
+        in_memory_zip.getvalue(),
+        content_type='application/zip',
+        direct_passthrough=True
+    )
```

Comment:
It might be good to add the following headers:
- `content_length`
- `Content-Disposition`

Similar to how it is applied here (https://github.com/superdesk/sams/blob/develop/src/server/sams/api/consume/assets.py#L54). I don't think there is any need for the `_etag` or `caching` headers